content/docs/command-reference/data/status.md

```diff
@@ -90,11 +95,9 @@ $ dvc data status
 Not in cache:
   (use "dvc fetch <file>..." to download files)
         data/data.xml
-
```

Comment:
Whitespace is significant.

Reply:
Not sure how this was deleted. Maybe linter or just a typo. Restoring whitespaces!